### ISSUE TITLE
looking forward to your code

### ISSUE THREAD

**Issue by mosu027** [tushar-semwal/TransferLearning_CNN_TextClassification#1]
Can I ask when the code will be released ?
thanks, thanks ,thanks!

**Comment by SeekPoint** [comment]
waiting.....

**Comment by jithurjacob** [comment]
waiting............

**Comment by mosu027** [comment]
waiting......

**Comment by tushar-semwal** [comment]
Thank you for the interest. Since this paper was part of my work in Samsung, the code needs to get reviewed. Hopefully, the code will be uploaded in the first week of next month.
FYI: The library used is Theano.

**Comment by SeekPoint** [comment]
Theano is deprecated.

**Comment by tushar-semwal** [comment]
Yup. I know! The work  was compiled at the time (around Sept.) Theano was still living. 
Though I still love Theano!

**Comment by tutubalinaev** [comment]
waiting too...

**Comment by don-tpanic** [comment]
waiting....

**Comment by ChaooMa** [comment]
waiting...

**Comment by xuxiaoyuspallow** [comment]
waiting...

**Comment by faysalhossain2007** [comment]
Would love to see your implementation.

**Comment by nickluijtgaarden** [comment]
Hi, for my current research I am lookinto transfer learning applied to abstractive summarization.

Can you share your code? I am really interested in it.

**Comment by FreeLoopBA** [comment]
waiting,thanks

**Comment by foreversolar** [comment]
waiting...
